recipes/liburing/all/patches/disable_samples_and_tests.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/Makefile b/Makefile
+index 73d021c..08ebd96 100644
+--- a/Makefile
++++ b/Makefile
+@@ -7,9 +7,9 @@ INSTALL=install
+ default: all
+ 
+ all:
+-	@$(MAKE) -C src
+-	@$(MAKE) -C test
+-	@$(MAKE) -C examples
++	@$(MAKE) -C src liburing.a
++	#@$(MAKE) -C test
++	#@$(MAKE) -C examples
+ 
```

Comment:
Minor: For a cleaner diff, could just remove the two bottom lines

Reply:
fixed